_episodes/12-export-transformation.md

```diff
@@ -1,16 +1,24 @@
 ---
-title: "Transformations - Exporting"
+title: "Exporting your revised data"
 teaching: 5
 exercises: 0
 questions:
-- "How do I export transformed data?"
+- "How do I export data from OpenRefine?"
 objectives:
 - "Explain how to export data in different formats from OpenRefine"
 keypoints:
 - "You can export your data in a variety of formats"
 ---
 
+## Note about OpenRefine
+All the edits you make to your data using OpenRefine are being stored inside the OpenRefine program and are not being saved to your original file. That's why OpenRefine uses the terms "import" and "export" to talk about moving your data in and out of the OpenRefine interface. Therefore, in order to save your work into a file format you can view in other programs or share with others, you need to export your data.
+
 ## Exporting data
 Once you have finished working with a data set in OpenRefine you may wish to export it. The export options are accessed through the ```Export``` button at the top right of the OpenRefine interface.
 
 Export formats support include HTML, Excel and comma- and tab-separated value (csv and tsv). You can also write a custom export, selecting to export specific fields, adding a header or footer and specifying the exact format.
```

Comment:
Hope it's appropriate to suggest this here but would the phrase

> Export formats support include HTML, Excel and comma- and tab-separated value (csv and tsv).

work better as

> Export formats support include HTML, Excel and csv and tsv (comma- and tab-separated value).

This would make it clearer that the "-" here is a hyphen indicating an word fragment rather than a dash.